sh/tests/solutions/in-back-ground.sh

```diff
@@ -0,0 +1 @@
+nohup cat facts | grep "moon" && echo "The moon fact was found!" >sol-output.log 2>&1 &
```

Comment:
the `2>&1` is not affecting the final test result - submitting a solution without this will pass the tests

Reply:
removed